src/handlers/RF24StreamLogHandler.cpp

```diff
@@ -0,0 +1,217 @@
+/**
+ * @file RF24StreamLogHandler.cpp
+ *
+ * Created on: 2 Oct 2020
+ *     Author: Witold Markowski (wmarkow)
+ *
+ * Copyright (C)
+ *    2020        Witold Markowski (wmarkow)
+ *
+ * This General Public License does not permit incorporating your program into
+ * proprietary programs.  If your program is a subroutine library, you may
+ * consider it more useful to permit linking proprietary applications with the
+ * library.  If this is what you want to do, use the GNU Lesser General
+ * Public License instead of this License.
+ */
+#include <Arduino.h>
+
+#include "RF24StreamLogHandler.h"
+
+const char rf24logLevelError[] = "ERROR";
+const char rf24logLevelWarn[] = " WARN";
+const char rf24logLevelInfo[] = " INFO";
+const char rf24logLevelDebug[] = "DEBUG";
+const char rf24logLevelTrace[] = "TRACE";
+
+const char *const rf24LogLevels[] = {rf24logLevelError,
+                                     rf24logLevelWarn,
+                                     rf24logLevelInfo,
+                                     rf24logLevelDebug,
+                                     rf24logLevelTrace};

```

Comment:
I fail to see how the end user would be able to customize these without completely redefining them when "implementing their own handler".

Reply:
"redefining them when" is the only way to go. Actually he needs to copy the content of the **RF24StreamLogHandler.cpp** and create a new file with custom body. Yeah, I know - it looks like some kind of code duplication, but the file is not so big and full customization is achieved:
```
const char rf24logLevelError[] = "E";
const char rf24logLevelWarn[] = " W";
const char rf24logLevelInfo[] = " I";
const char rf24logLevelDebug[] = "D";
const char rf24logLevelTrace[] = "T";

const char *const rf24LogLevels[] = {rf24logLevelError,
                                     rf24logLevelWarn,
                                     rf24logLevelInfo,
                                     rf24logLevelDebug,
                                     rf24logLevelTrace};
```